src/tests/Interop/UnmanagedCallersOnlyBasic/UnmanagedCallersOnlyBasicTest.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Threading;
+using Xunit;
+
+public unsafe class Program
+{
+    public static class UnmanagedCallersOnlyDll
+    {
+        [DllImport(nameof(UnmanagedCallersOnlyDll))]
+        public static extern int CallManagedProc_Stdcall(delegate* unmanaged[Stdcall]<int, int> callbackProc, int n);
+
+        [DllImport(nameof(UnmanagedCallersOnlyDll))]
+        public static extern int CallManagedProc_Cdecl(delegate* unmanaged[Cdecl]<int, int> callbackProc, int n);
+    }
+
+    public static int Main(string[] args)
+    {
+        var result = 100;
+
+        result += TestUnmanagedCallersOnlyValid_CallConvStdcall();
+        result += TestUnmanagedCallersOnlyValid_CallConvCdecl();
+
+        return result;
+    }
```

Comment:
Remove `Main` and place `[Fact]` on all test methods. We have as a source generator that will generate `Main` for you.

Reply:
Do you want me to do the same for the `UnmanagedCallersOnlyTest`?